lib/css/modules/_userTagger.scss

```diff
@@ -111,6 +111,11 @@ a.voteWeight {
 
 .RESUserTag {
 	color: black;
+
+	// unselectable in comments/markdown for usability
+	.md & {
```

Comment:
I wonder how long until somebody asks for this back.. but hey, we can give them a CSS snippet or a bodyclass or something.